<launch>

  <node name="mask_service" pkg="toy_assembly" type="ada_client.py" output="screen" />

  <node name="transcribe" pkg="toy_assembly" type="audio.py" output="screen" >
    <param name="threshold"  value="0.0510" />
    <param name="silent_wait" value="5" />
    <param name="max_duration"  value="25" />
  </node>

</launch>